BannedScript/banScript.py

```diff
@@ -0,0 +1,60 @@
+import yaml
+import pymongo
+import requests
+
+# Load YAML file
+with open('./config/config.yml', 'r') as file:
+    config = yaml.safe_load(file)
+
+authToken = input("Please provide your token: ")
```

Comment:
instead of this add command line arguments here. https://www.tutorialspoint.com/python/python_command_line_arguments.htm